recipes/xmscore/meta.yaml

```diff
@@ -1,27 +1,26 @@
-{% set name = "xmscore" %}
 {% set version = "3.0.7" %}
-{% set sha256 = "98acd332ac5a14602539e218db82b2970dbeed2b149e0d47dcbd80c6778801c7" %}
 
 package:
-  name: {{ name|lower }}
+  name: xmscore
   version: {{ version }}
 
 source:
-  url: https://github.com/Aquaveo/{{ name|lower }}/archive/{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://github.com/Aquaveo/xmscore/archive/{{ version }}.tar.gz
+  sha256: 98acd332ac5a14602539e218db82b2970dbeed2b149e0d47dcbd80c6778801c7
 
 build:
   number: 0
+  skip: True  # [not py36]
+  merge_build_host: true  # [linux]
```

Comment:
This will help with the packages that should be on host but the build system is expecting them on `build`. I believe we can drop this if we fix the build makefile. Not worth doing that now.

Reply:
We may need to add this to Windows and macOS too. Let's wait for the CIs to be sure.